src/CameraProps.ts

```diff
@@ -156,6 +156,11 @@ export interface CameraProps extends ViewProps {
    * @default false
    */
   enableHighQualityPhotos?: boolean;
+  /**
+   * Enable the Camera to take photos with RAW format.
+   * @platform iOS 10+
+   */
+  enableRawCapture?: boolean;
```

Comment:
I think instead of a boolean it makes more sense to have a union type here so the user knows it's an "either this or that".

Something like

`captureType: 'raw' | 'photo'`

Maybe a better name for 'photo'.